cmd/gokr-packer/packer.go

```diff
@@ -466,11 +475,15 @@ func verifyNotMounted(dev string) error {
 	return nil
 }
 
-func (p *pack) overwriteDevice(dev string, root *fileInfo) error {
+func (p *pack) overwriteDevice(dev string, root *fileInfo, rootDeviceFiles []deviceconfig.RootFile) error {
 	if err := verifyNotMounted(dev); err != nil {
 		return err
 	}
-	log.Printf("partitioning %s (GPT + Hybrid MBR)", dev)
+	parttable := "GPT + Hybrid MBR"
+	if !p.UseGPT {
+		parttable = "MBR"
```

Comment:
To make it really clear to readers what’s happening, let’s change this from “MBR” to “no GPT, only MBR”